src/main/java/hudson/plugins/analysis/graph/GraphConfiguration.java

```diff
@@ -433,6 +493,8 @@ private void reset() {
         dayCount = DEFAULT_DAY_COUNT;
         graphType = DEFAULT_GRAPH;
         useBuildDate = DEFAULT_USE_BUILD_DATE;
+        parameterName = null;
```

Comment:
null better indicates no value set. Is there a specific reason for this to be empty string? Would it become empty string if this value came from GUI configuration?